lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -0,0 +1,187 @@
+define((require, exports, module) ->
+  Behaviour = require("ace/mode/behaviour").Behaviour
+
+  isCommentToken = (token) -> token? and /comment/.test(token.type)
```

Comment:
Also, shall it really be regexp, not string equals, or starts with, or substring search?